code/__DEFINES/role_preferences.dm

```diff
@@ -108,7 +108,6 @@
 #define ROLE_BATTLECRUISER_CAPTAIN "Battlecruiser Captain"
 #define ROLE_VENUSHUMANTRAP "Venus Human Trap"
 
-
```

Comment:
don't remove

Reply:
I'm confused i was told to remove that?

Reply:
don't remove the random line of whitespace

Reply:
OH I didnt realize i did that